demo-notebooks/batch-job/batch_mnist_mcad.ipynb

```diff
@@ -0,0 +1,3376 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "b55bc3ea-4ce3-49bf-bb1f-e209de8ca47a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Import pieces from codeflare-sdk\n",
+    "from codeflare_sdk.cluster.auth import TokenAuthentication"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "9de86658",
+   "metadata": {},
+   "source": [
+    "First, we begin by authenticating using the SDK."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "614daa0c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create authentication object for oc user permissions\n",
+    "auth = TokenAuthentication(\n",
+    "    token = \"XXXXX\",\n",
+    "    server = \"XXXXX\",\n",
+    "    skip_tls=True\n",
+    ")\n",
+    "auth.login()"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "87d2c9b3",
+   "metadata": {},
+   "source": [
+    "Now that we are logged in, we can directly submit our batch job (model training on two workers with four gpus each) to MCAD via torchx."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "3cc6183a-8f6e-4347-af91-d088ed422544",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from codeflare_sdk.job.jobs import DDPJobDefinition"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "ced6ccd6-a17e-413a-a0e4-65004fc35463",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": []
+    }
+   ],
+   "source": [
+    "job = DDPJobDefinition(name=\"mnistjob\", script=\"mnist.py\", scheduler_args={\"namespace\": \"default\"}, j=\"1x1\", gpu=0, cpu=1, memMB=8000, image=\"quay.io/michaelclifford/mnist-test:latest\").submit()"
```

Comment:
Can we reformat this so that its vertically aligned? 
```suggestion
    "job = DDPJobDefinition(name=\"mnistjob\", 
    script=\"mnist.py\", 
    scheduler_args={\"namespace\": \"default\"}, 
    j=\"1x1\", 
    gpu=0, 
    cpu=1, 
    memMB=8000, 
    image=\"quay.io/michaelclifford/mnist-test:latest\").submit()"
```